es5/videojs-dailymotion.js

```diff
@@ -66,7 +66,7 @@ var Dailymotion = (function (_Tech) {
 
     if (typeof this.videoId !== 'undefined') {
       this.setTimeout(function () {
-        _this.setPoster('//api.dailymotion.com/video/' + _this.videoId + '?fields=poster_url&ads=false');
+        _this.setPoster('https://api.dailymotion.com/video/' + _this.videoId + '?fields=thumbnail_large_url');
```

Comment:
I think it'll be better to stick to `//api.` rather than enforcing `https://`, that way it'll ensure compatibility from all sort of requests. Don't you think so?